tool/ood-gen/lib/event.ml

```diff
@@ -0,0 +1,64 @@
+type location = { lat : float; long : float }
+[@@deriving of_yaml, show { with_path = false }]
+
+type date_with_optional_time = Date of Ptime.date | DateTime of Ptime.t
+
```

Comment:
```suggestion
```

Since you're not using this type, let's remove it. We can still refine this in another pull request later.